### DISCUSSION TITLE
unhandledRejection: TypeError: Cannot read properties of null (reading 'useReducer')

### DISCUSSION THREAD

**Discussion by Kamadarada** [vercel/next.js#61630] [Help] [answered]
### Summary

Hey guys!

i started my studies with next.js and i got this error when i'm trying to run the application.

"unhandledRejection: TypeError: Cannot read properties of null (reading 'useReducer')" 
but i'm not using "useReducer" in any archive.

already tried to downgrade react 

![image](https://github.com/vercel/next.js/assets/112417899/af7cd7dd-e285-4c53-af73-ba359edf6860)
![image](https://github.com/vercel/next.js/assets/112417899/4956b835-a0bc-44d3-b7f6-ee1cd14a8092)

**Comment by Kamadarada** [comment, marked as answer]
Here is how i solved it:

- yarn set version stable 
- yarn install 
- yarn cache clean 
- yarn dev

apparently it was just the yarn cache

**Comment by khokonm** [comment]
I'm using NPM, still facing the issue on newly created nextjs project, how do I solve?

**Reply by Kamadarada** [reply]
the problem was yarn cache. So try clearing npm cache

**Reply by akhilravidas** [reply]
```
npm install -g npm@latest
npm install
npm cache clean --force
npm run dev
```

**Reply by khokonm** [reply]
Turns out, the issue was specific to my device due to how I initially set up Next.js. After almost eight months, I finally fixed the issue and was able to run a Next.js project again. If anyone else is facing the same problem, feel free to check out the detailed documentation I wrote on [how I fixed the issue!](http://khokon.dev/weekup/blog/nextjs/01-fix-TypeError-Cannot-read-properties-of-null-reading-useReducer).

**Reply by Vancelott** [reply]
@khokonm Your blog post saved me after trying to debug this error for two days, thanks a ton!
